cltool/src/cltool_main.cpp

```diff
@@ -714,6 +714,12 @@ static int cltool_dataStreaming()
         cout << "Failed to open serial port at " << g_commandLineOptions.comPort.c_str() << endl;
         return -1;    // Failed to open serial port
     }
+    g_inertialSenseDisplay.SetSerialPort(inertialSenseInterface.SerialPort());
```

Comment:
This seems like a strange place (`cltool_requestDataSets()`) to be doing ISDisplay setup.  Should be be moved into `cltool_streamData()` or `cltool_setupCommunications()`?

Reply:
Moved into cltool_setupCommunications()